src/Evolution/Systems/GeneralizedHarmonic/Characteristics.cpp

```diff
@@ -65,13 +57,15 @@ void compute_characteristic_fields(
     const gsl::not_null<typename Tags::CharacteristicFields<Dim, Frame>::type*>
         char_fields,
     const Scalar<DataVector>& gamma_2,
+    const tnsr::II<DataVector, Dim, Frame>& inverse_spatial_metric,
     const tnsr::aa<DataVector, Dim, Frame>& spacetime_metric,
     const tnsr::aa<DataVector, Dim, Frame>& pi,
     const tnsr::iaa<DataVector, Dim, Frame>& phi,
-    const tnsr::i<DataVector, Dim, Frame>& unit_normal_one_form,
-    const tnsr::I<DataVector, Dim, Frame>& unit_normal_vector) noexcept {
+    const tnsr::i<DataVector, Dim, Frame>& unit_normal_one_form) noexcept {
   auto phi_dot_normal =
       make_with_value<tnsr::aa<DataVector, Dim, Frame>>(pi, 0.);
+  auto unit_normal_vector =
+      raise_or_lower_index(unit_normal_one_form, inverse_spatial_metric);
```

Comment:
I had thought the whole reason for requiring the unit_normal_vector as well as unit_normal_one_form as inputs was to avoid duplicating the raise_or_lower_index cost...is there a reason to go back to computing this inside compute_characteristic_fields?

Reply:
We tag the normal_one_form with `UnnormalizedFaceNormal`, but there doesn't appear to be a tag for the equivalent `UnnormalizedFaceNormalVector` and therefore I removed its need. I am happy to add it back if we also want to add a `UnnormalizedFaceNormalVector` tag, but that may be best done in its own PR.